### ISSUE TITLE
Question: can I use this if my repo is not a fork?

### ISSUE THREAD

**Issue by linonetwo** [exions/merge-upstream#8]
Can I merge arbitrary public repo to my repo?

Can I merge several repos to a sub directory of my mono repo?

Thanks

**Comment by Qubik65536** [comment]
Yes, you will be able to use this even if your repo is not a fork. I haven't tried to merge multiple repos to sub directory, but I think you will be able merge different repos to different branches.
